iosHackthonApp/Modules/Features/MealList/Sources/MealList/MealListViewModel.swift

```diff
@@ -33,14 +33,16 @@ public final class MealListViewModel: ObservableObject {
     }
 
     func loadMeals(forceReload: Bool) {
-        sdk.getMeals(forceReload: forceReload, completionHandler: { meals, error in
+        sdk.getSortedMeals(userLat: locationManager.userLatitude,
+                           userLon: locationManager.userLongitude,
+                           distanceUnit: 0,
```

Comment:
Will change this.
Might have a look at putting the enum for this distanceUnit into the shared code 🤔